src/controllers/bulkUpload.ts

```diff
@@ -82,7 +82,7 @@ export const pollStatus = async (ctx) => {
     JSON.stringify(ctx.request.body),
   );
 
-  const { destType } = ctx.request.body;
+  const { destType }: { destType: string } = ctx.request.body;
```

Comment:
[minor] Define as a separate type & use it.

Reply:
Done